### PR TITLE
Framework: Remove deprecations slated for 2.5.0 removal

### PR DESCRIPTION
**Pull request by aduth**
This pull request seeks to remove deprecations slated for removal in the upcoming 2.5.0 release.

Specifically, this includes:

- Returning raw HTML string from block `save`
- `wp.data.query`

`wp.blocks.Editable` was slated for removal as well, but since the [Editable API](https://wordpress.org/gutenberg/handbook/block-api/editable-api/) is still (mistakenly) documented on the Gutenberg handbook, I'm inclined to continue to push this to being removed in a future release.

This also updates a the unwrapped paragraph deprecation approach implemented in #4874 to correctly use `RawHTML` in the deprecated `save`. This issue became obvious with the removal thanks to existing fixture tests.

__Testing instructions:__

Verify that there are no regressions in impacted behavior, and importantly that no references to deprecated behaviors exist in core code.

### CODE REVIEW

File: blocks/library/paragraph/index.js [hunk 1]
```diff
@@ -389,7 +389,7 @@ export const settings = {
 				},
 			},
 			save( { attributes } ) {
-				return attributes.content;
+				return <RawHTML key="html">{ attributes.content }</RawHTML>;
```
Comment (youknowriad) [inline blocks/library/paragraph/index.js]: Is key necessary here?

File: blocks/rich-text/editable.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ class Editable extends RichText {
 	constructor() {
 		super( ...arguments );
 		deprecated( 'Editable', {
-			version: '2.5',
+			version: '2.6',
```
Comment (youknowriad) [inline blocks/rich-text/editable.js]: Should we change it in `deprecated.md` as well?

File: blocks/library/paragraph/index.js [hunk 3]
```diff
@@ -389,7 +389,7 @@ export const settings = {
 				},
 			},
 			save( { attributes } ) {
-				return attributes.content;
```
Comment (youknowriad) [inline blocks/library/paragraph/index.js]: We still have an instance of string saving in the parser's unit test but I'm certain it's used at all.

File: blocks/library/paragraph/index.js (hunk 1 above)
Comment (aduth) [reply blocks/library/paragraph/index.js]: >Is key necessary here?

Nope. Mistaken copy from the `migrate` function.

File: blocks/rich-text/editable.js (hunk 2 above)
Comment (aduth) [reply blocks/rich-text/editable.js]: >Should we change it in `deprecated.md` as well?

Yep. Added.

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Pull request merged**